deploy/markets.ts

```diff
@@ -129,6 +130,7 @@ const initializeMarkets: DeployFunction = async (hre) => {
                 case 'TokenSymbol':
                   return tokenAddresses.all[value]
                 case 'Address':
+                  return dappAddresses[value]
```

Comment:
this is good for now but will probably have to rethink things when we want to use addresses for other things

Reply:
can add additional types? `AddressProviderConstant` as an additional type

Reply:
Added an additional type `ProtocolAddressConstant` for now